stufftodo.txt

```diff
@@ -0,0 +1,3 @@
+1. are we going to use Systick for both motor moves
+2. are we planning on moving both motors at the same time
+3. how to stop the motor when it is done moving.
```

Comment:
Since the drivers have a step-direction two-pin interface, we simply stop toggling the step bit and it will hold its position.